### ISSUE TITLE
connect CircleCI to VotingWorks slack

### ISSUE THREAD

**Issue by danwallach** [votingworks/arlo-e2e#4]

**Comment by platipo** [comment]
Hi @danwallach, what do you mean with "connect"? Sending a notification to VotingWorks on the build status?

**Comment by danwallach** [comment]
Yeah, all the other repos generate Slack notifications from CircleCI. This is hanging out on my to-do list.

**Comment by platipo** [comment]
Notification should be triggered on success and on failure or only on failure?
Environment variables are already set for Slack?

**Comment by danwallach** [comment]
This is really an internal VotingWorks issue. Not something that I want/need external help resolving.
